src/jit/compiler.cpp

```diff
@@ -540,7 +540,7 @@ var_types    Compiler::argOrReturnTypeForStruct(unsigned size, CORINFO_CLASS_HAN
         break;
 
     default:
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET && !defined(_TARGET_X86_)
```

Comment:
If you don't change this, you need to update the corresponding `#endif` comment.